stage1/reaper/reaper.sh

```diff
@@ -7,5 +7,11 @@ if [ $# -eq 1 ]; then
     app=$1
     status=$(${SYSCTL} show --property ExecMainStatus "${app}.service")
     echo "${status#*=}" > "/rkt/status/$app"
+    if [ "${status#*=}" != 0 ] ; then
+        # The command "systemctl exit $status" is available since systemd v227.
+        # On older versions, the command will fail and rkt will just exit with
+        # return code 0.
+        ${SYSCTL} exit ${status#*=}
+    fi
```

Comment:
Please also say in the command that `systemctl exit` does not actually exit the pod right now, but merely update the return value in systemd that will be used when the pod exits later. The pod actually exits via `shutdown.service`.

Is it correct?